pkg/backup/restore/restore_manager.go

```diff
@@ -66,6 +69,80 @@ func (rm *restoreManager) UpdateCondition(restore *v1alpha1.Restore, condition *
 	return rm.statusUpdater.Update(restore, condition, nil)
 }
 
+func (rm *restoreManager) pitrEnable(tc *v1alpha1.TidbCluster) error {
+	if tc.Status.TiKV.PiTRStatus.Active {
+		return nil
+	}
+	s := v1alpha1.ConfigUpdateStrategyInPlace
+	tc.Spec.TiKV.ConfigUpdateStrategy = &s
```

Comment:
should we reset this ConfigUpdateStrategy back after Update finished?

Reply:
This might cause TiKV restart if we do so?